Neos.Media.Browser/Resources/Private/Translations/da/Main.xlf

```diff
@@ -68,6 +68,9 @@
       <trans-unit id="editAsset" xml:space="preserve" approved="yes">
 				<source>Edit asset</source>
 			<target xml:lang="da">Rediger fil</target><alt-trans><target xml:lang="da">Rediger</target></alt-trans></trans-unit>
+      <trans-unit id="replaceAssetResource" xml:space="preserve" approved="yes">
```

Comment:
All these files are managed by crowdin, or am I wrong? So they shouldn't be altered manually.